test/unit/helpers/mocks/Util.js

```diff
@@ -91,18 +91,22 @@ function ($, _, Backbone, Q, Duo) {
       }
 
       var deferred = $.Deferred();
-      if (xhr.status > 0 && xhr.status < 300) {
-        // $.ajax send (data, textStatus, jqXHR) on success
+
+      (function (textOnly) {
```

Comment:
Right, but what was the problem with the previous code? I see the difference is that no matter with ```xhr.status``` we have, now all that code is inside a ```setTimeout```. Why do we need to change that?